### PR TITLE
Feat: <Table> component preview

### PR DESCRIPTION
**Pull request by juanesquintero**

### CODE REVIEW

**Comment by juanesquintero** [comment]
Hey @wyozi  let me know if theese styles work for <Table> preview... :)

File: pages/index.tsx [hunk 1]
```diff
@@ -166,6 +167,30 @@ const code = `ReactPPTX.render(
...
+        style={{
+          x: "10%",
+          y: "10%",
+          w: "80%",
+          h: "80%",
+          borderWidth: 10,
+          borderColor: "#ff0000",
+        }}
+      />
+</Slide>
```
Comment (wyozi) [inline pages/index.tsx:193]:
```suggestion
    </Slide>
```

**Comment by wyozi** [comment]
Seems good! I'll merge with the spacing fixed

**Comment by wyozi** [comment]
Thanks 🚀

**Comment by juanesquintero** [comment]
@wyozi Great! thanks for the quick response and release launch!!🚀  Really happy to contribute more to the project!

**Pull request merged**